### ISSUE TITLE
Get notifications when an integration fails

### ISSUE THREAD

**Issue by Mosnar** [verbb/formie#1829]
### What are you trying to do?

We have a Salesforce integration setup that occasionally fails due to duplicate submissions. The client would like to receive notifications in their email when this happens so they can address it.

### What's your proposed solution?

Add a way to get notifications when integrations fail for whatever reason. Bonus points for being able to review these failures in a central list rather than the queue.

**Comment by engram-design** [comment]
Agreed, and we're either going to address this directly within Formie to send out emails when something fails, or create another (free) plugin that's generic to when _anything_ in the queue fails. We're thinking the latter might be more useful, but can understand the hesitation with installing another plugin for this.

As for a central list, that's potentially an option, but we'd just be re-creating the queue screen that's all.

**Comment by engram-design** [comment]
Going to mark this as "done" with the help of the [Queue Monitor](https://github.com/verbb/queue-monitor) plugin.
